### PR TITLE
Gotta get me those avgs

### PR DESCRIPTION
**Pull request by PaulIvanov**
YTD data stuff on metric view and ability to change the pay period from the pay period view

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -238,6 +241,8 @@ <h5 v-text="metricsTitle"></h5>
 							<div class="metrics-chart-wrapper">
 								<canvas id="metrics-chart"></canvas>
 							</div>
+							<div>
```
Comment (jomogalla) [inline index.html]:
```suggestion
				
```

File: index.html [hunk 2]
```diff
@@ -238,6 +241,8 @@ <h5 v-text="metricsTitle"></h5>
...
+							</div>
```
Comment (jomogalla) [inline index.html]:
```suggestion
							
```

File: scripts/app.js [hunk 3]
```diff
@@ -584,6 +598,15 @@
 				utilities.loader.start();
 				utilities.api(listIntervalsForDate).then(this.handlePayPeriodRequest);
 			},
+			goToPreviousPayPeriod: function () {
+				let dates = this.getPayPeriodRange(this.$_currentPayPeriod);
+				this.getPayPeriod(dates[0].subtract(1, "days"));
+			},
+			goToNextPayPeriod: function () {
+				let dates = this.getPayPeriodRange(this.$_currentPayPeriod);
+				this.getPayPeriod(dates[1].add(11, "days"));
```
Comment (jomogalla) [inline scripts/app.js]: Is 11 just an arbitrary number to put the date into the next pay period?

**Review by jomogalla — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (jomogalla) [reply index.html]: Removed these useless looking divs... and tried some new suggested change feature

File: scripts/app.js (hunk 3 above)
Comment (PaulIvanov) [reply scripts/app.js]: Looking at this again, I have no idea. TBF I was buzzed up when I did this. ill get back to you on this

Comment (PaulIvanov) [reply scripts/app.js]: @jomogalla I actually think it was meant to be a 1. dates[1] returns the endDate of a payPeriod. Incrementing it by 1 should give you the start of the next pay period.

**Review by PaulIvanov — COMMENTED** [review]
Think its good now

**Review by jomogalla — APPROVED** [review]

**Pull request merged**